common/web/types/src/ldml-keyboard/ldml-keyboard-xml-reader.ts

```diff
@@ -1,3 +1,6 @@
+// Enables DOM types, but just for this one module.
+///<reference lib="dom" />
+
```

Comment:
`/common/web/keyboard-processor/build.sh build` will run into a build error within this file if this line is not present.  It compiles without access to the DOM libraries except where made explicit.

Reply:
Hmm, @srl295 I think we should pull `URL` usage out of this module and have a helper instead. It's apparently used in 4 places altogether in unit tests and kmc:

* kmc buildTestData/index.ts
* kmc BuildLdmlKeyboard.ts
* unit test: reader-callback-test.ts
* unit test helpers/index.ts

Propose the following:

```
static get defaultImportsURL() {
  return ['../import/', import.meta.url];
}
```

And in usage:

```
    readerOptions: {
      importsPath: fileURLToPath(new URL(...LDMLKeyboardXMLSourceFileReader.defaultImportsURL))
    }
```

Reply:
I presume that'd be a separate issue, not something I ought do here?

Reply:
Yeah it could be a separate PR. Be good to address it before this change lands in master though, just for cleanliness.

Reply:
Up as #9788.